src/url-replacements.js

```diff
@@ -88,6 +88,17 @@ class UrlReplacements {
       return url && url.hostname;
     });
 
+    // Returns the Source URL for this AMP document.
+    this.set_('SOURCE_URL', () => {
+      return removeFragment(getSourceUrl(this.win_.location.href));
+    });
+
+    // Returns the host of the Source URL for this AMP document.
+    this.set_('SOURCE_HOST', () => {
```

Comment:
All other URLs have X_HOST already, unfortunately. I think we should just stay consistent.